src/main.rs

```diff
@@ -420,11 +421,25 @@ fn run() -> Result<ExitStatus> {
         }
     }
 
-    eprintln!("Warning: Falling back to `cargo` on the host.");
-
     // if we fallback to the host cargo, use the same invocation that was made to cross
     let argv: Vec<String> = env::args().skip(1).collect();
-    cargo::run(&argv, verbose)
+    eprintln!("Warning: Falling back to `cargo` on the host.");
+    match args.subcommand {
+        Some(Subcommand::List) => {
+            // TODO: this won't print in order if we have both stdout and stderr.
+            let (status, stdout) = cargo::run_and_get_status_and_stdout(&argv, verbose)?;
```

Comment:
I don't understand this todo, it reads more like a general comment

Reply:
Ah true, there's probably no way to fix it either since writing to separate buffers isn't guaranteed to be in order. I'll remove the TODO.